examples/react-todos/.meteor/packages

```diff
@@ -13,5 +14,6 @@ reactive-var
 meteorhacks:npm
 cosmos:browserify
 
+static-html
```

Comment:
Well, there is always room for improvement! We could minify the HTML if necessary.